rmgpy/rmg/main.py

```diff
@@ -692,6 +700,24 @@ def execute(self, **kwargs):
         
         self.finish()
 
+    def generateCanteraFiles(self, chemkinFile, **kwargs):
+        """
+        Convert a chemkin mechanism chem.inp file to a cantera mechanism file chem.cti
+        and save it in the cantera directory
+        """
+        transportFile = os.path.join(os.path.dirname(chemkinFile), 'tran.dat')
+        fileName = os.path.splitext(os.path.basename(chemkinFile))[0] + '.cti'
+        outName = os.path.join(self.outputDirectory, 'cantera', fileName)
+        canteraDir = os.path.dirname(outName)
+        try:
+            os.stat(canteraDir)
+        except:
+            os.mkdir(canteraDir)
```

Comment:
An alternative way to only create a directory if it doesn't exist would be to try to create it first, then catch if it fails:
```
try: 
    os.makedirs(path)
except OSError:
    if not os.path.isdir(path):
        raise
```
This will directly create the directory when it doesn't exist (most cases), do nothing if the directory exists, and raise an error if os.mkdir fails for other reasons (eg. if a file exists with the same name).

This method will also prevent a "race condition" where an error can occur if the directory is somehow created in between the call to os.stat and the call to os.mkdir.